packages/x-utils/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: 'tsconfig.eslint.json'
+  },
+  overrides: [
```

Comment:
I've done this to avoid having to disable in each `.types.spec.ts` file the `@typescript-eslint/ban-ts-comment` rule to be able to use `@ts-expect-error` for _type-checking_